samples/core/xgboost_training_cm/xgboost_training_cm.py

```diff
@@ -101,24 +96,40 @@ def dataproc_transform_op(
     analysis,
     output
 ):
-    return dsl.ContainerOp(
-        name='Dataproc - Transform',
-        image='gcr.io/ml-pipeline/ml-pipeline-dataproc-transform:57d9f7f1cfd458e945d297957621716062d89a49',
-        arguments=[
-            '--project', project,
-            '--region', region,
-            '--cluster', cluster_name,
-            '--train', train_data,
-            '--eval', eval_data,
-            '--analysis', analysis,
-            '--target', target,
-            '--output', output,
-        ],
-        file_outputs={
-            'train': '/output_train.txt',
-            'eval': '/output_eval.txt',
-        }
-    )
+  """Submit dataproc transform as a pyspark job.
+
+  :param project: GCP project ID.
+  :param region: Which zone to run this analyze.
+  :param cluster_name: Name of the cluster.
+  :param train_data: GCS path to the training data.
+  :param eval_data: GCS path of the eval csv file.
+  :param target: Target column name.
+  :param analysis: GCS path of the analysis results
+  :param output: GCS path to use for output.
+  """
+
+  # Remove existing [output]/train and [output]/eval if they exist.
+  delete_directory_from_gcs(os.path.join(output, 'train'))
```

Comment:
Are you sure this is going to work?

`delete_directory_from_gcs` is not a component, so it won't be executed as part of the pipeline.